spec/index.md

```diff
@@ -3409,6 +3486,42 @@ if A.settings.controllers is not null:
 else:
   New_controllers = [M.caller]
 
+if New_compute_allocation > 0 or New_memory_allocation > 0:
```

Comment:
In principle, we could always check, but the implementation only checks in those cases.